ui/app/models/kmip/scope.js

```diff
@@ -16,5 +16,5 @@ export default Model.extend({
   attrs: computed(function () {
     return expandAttributeMeta(this, ['name']);
   }),
-  updatePath: lazyCapabilities(apiPath`${'backend'}/scope/${'id'}`, 'id'),
+  updatePath: lazyCapabilities(apiPath`${'backend'}/scope/${'id'}`, 'backend', 'id'),
```

Comment:
Whoops! good catch

Reply:
I should have caught it in the code review! I even thought to check for each arg was present but missed this one 😞 

This was failing a test for me, so I wonder if it was something I changed